src/mount/mount.ceph.c

```diff
@@ -309,9 +309,8 @@ static int parse_options(const char *data, struct ceph_mount_info *cmi)
 			name = value;
 			skip = false;
 		} else {
+			/* unrecognized mount options, passing to kernel */
 			skip = false;
-			mount_ceph_debug("mount.ceph: unrecognized mount option \"%s\", passing to kernel.\n",
-					data);
```

Comment:
Yes, I think we want to keep the message. I just want to stop seeing this message for `mds_namespace`.

Reply:
Then don't request verbose messages....I'm not clear on what problem this is solving given that this involves the user requesting verbose messages

Reply:
My problem with this message is that `mds_namespace` is not "unrecognized". It is a known option that we just pass to the kernel. I don't want to see a message that looks like an error with a typical mount option.

Reply:
Well, it _is_ unrecognized by the mount helper. If you want to rephrase what it says, then I'm ok with that. The important bit is that it didn't recognize that option and passed it through to the kernel. If you subtly fat finger a different option that you expect mount.ceph to recognize, it's good to see that message.

OTOH, I will grant that printing this is not really the most helpful way to pass info to the user. Maybe we _should_ get rid of this message, and add a new one just before the mount() call that shows the final option string (and maybe the flags).

Reply:
Yeah, I think to show a final option string is a good idea.

Reply:
Have done of this. It will be like:

```
# ./bin/mount.ceph :/ /mnt/cephfs/ -v -o fs=a,noacl
parsing options: fs=a,noacl
mount.ceph: options "mds_namespace=a,noacl" will pass to kernel.

```